modules/react/tooltip/spec/Tooltip.spec.tsx

```diff
@@ -28,10 +28,9 @@ describe('Tooltip', () => {
       fireEvent.mouseEnter(screen.getByText('Test Text')); // triggers the tooltip
 
       jest.advanceTimersByTime(300); // advance the timer by the amount of delay time
-      expect(screen.getByText('Test Text')).toHaveAttribute('aria-describedby');
+      expect(screen.getByText('Test Text')).toHaveAttribute('aria-description');
```

Comment:
I think it makes sense to assert the contents of the attribute and not just the presence of it. The second parameter is the text that is expected.